KubeArmor/build/build_kubearmor.sh

```diff
@@ -45,7 +45,7 @@ unset LABEL
 # build a kubearmor image
 DTAG="-t $REPO:$VERSION"
 echo "[INFO] Building $DTAG"
-cd $ARMOR_HOME/..; docker build $DTAG -f Dockerfile --target kubearmor . $LABEL
+cd $ARMOR_HOME/..; docker build $DTAG -f Dockerfile --target kubearmor . $LABEL -e ENABLE_PPROF=false
```

Comment:
the value should be handed down from the CI to this script, to the docker file , to the make file